### PR TITLE
Accessibility section in docs

### PR DESCRIPTION
**Pull request by jaketrent**
Brian, as I looked at this, I thought that the a11y module design I had made and what is accomplishable just with markdown were so close that I'd lean on the latter for the moment and see what we thought.

I think it's a fine experience. 

Two big reasons I like it: 1) It avoids having to deal with the API for a summary module. 2) Not having a set API allows for one-off descriptions of a11y as needed per component.

There are two commits. The first with the MDX capability. The second, giving an example on the Dropdown page. If I was you, and you end up liking this approach, I'd just copy the first commit into my own branch, ignore the 2nd commit and then close this PR when you're well on your way.

This is what it looks like rendered:
Screen Shot 2021-08-30 at 2 49 27 PM

### CODE REVIEW

**Review by sweeting — COMMENTED** [review]
Reviewed and made some modifications. @jaketrent take a look at the dropdown page and let me know what you think.

File: docs/src/components/mdx/index.tsx [hunk 1]
```diff
@@ -20,7 +25,10 @@ const components = {
   a: A,
   A,
   Badge,
+  Button,
   blockquote: BlockQuote,
+  CheckCircleFillIcon,
+  WarningFilledIcon,
```
Comment (jaketrent) [inline docs/src/components/mdx/index.tsx:31]: Please alpha order this list.

File: docs/src/components/usage.tsx [hunk 2]
```diff
@@ -1,27 +1,84 @@
...
-      <Item label="Install">{props.install.trim()}</Item>
-      <Item label="Import">{props.import.trim()}</Item>
-    </ul>
+    <>
+      <ul className={styles.links}>
+        <li>
+          <Button
+            appearance="secondary"
+            href={`https://github.com/pluralsight/design-system/blob/master/packages/${props.packageName}`}
+            target="_blank"
```
Comment (jaketrent) [inline docs/src/components/usage.tsx:26]: Please add companion attr `rel="noreferrer"` to these external links.

See https://web.dev/external-anchors-use-rel-noopener/

File: docs/src/components/usage.tsx [hunk 3]
```diff
@@ -1,27 +1,84 @@
...
-        <A
-          href={`https://github.com/pluralsight/design-system/blob/master/packages/${props.packageName}/CHANGELOG.md`}
-        >
-          {props.version || 'CHANGELOG'}
-        </A>
-      </Item>
-      <Item label="Install">{props.install.trim()}</Item>
-      <Item label="Import">{props.import.trim()}</Item>
-    </ul>
+    <>
```
Comment (jaketrent) [inline docs/src/components/usage.tsx:20]: Could remove this fragment tag.

File: docs/src/content/components/dropdown.mdx [hunk 4]
```diff
@@ -42,14 +50,23 @@ function Example() {
...
+      </div>
     </div>
   )
 }
 
 export default Example
 ```
 
-### Label
+### Label & Sublabels
```
Comment (jaketrent) [inline docs/src/content/components/dropdown.mdx:69]:
```suggestion
### Label & sublabels
```

File: docs/src/content/components/dropdown.mdx [hunk 5]
```diff
@@ -359,14 +343,52 @@ function Example() {
...
+    }
+  />
+)
+
+export default Example
+```
+
+## Accessibility
+
+**WCAG 2.1 AA Partial Compliance**
```
Comment (jaketrent) [inline docs/src/content/components/dropdown.mdx:384]: nbd, but I'd do something like this for these options:

- **Partially WCAG 2.1 AA Compliant**
- **WCAG 2.1 AA Compliant**

Why?

- Reads more like a status badge, applied to the component
- More easily scannable with "partial" at the front and looking different than full compliance

**Review by jaketrent — COMMENTED** [review]
Yeah, Brian, I can dig these changes. Great job!

Some suggestions inline.

Since it's my PR, GH is not allowing me to Approve. But, I do approve!

File: docs/src/components/usage.tsx (hunk 2 above)
Comment (sweeting) [reply docs/src/components/usage.tsx:26]: 👍

File: docs/src/components/mdx/index.tsx (hunk 1 above)
Comment (sweeting) [reply docs/src/components/mdx/index.tsx:31]: 👍

File: docs/src/components/usage.tsx (hunk 3 above)
Comment (sweeting) [reply docs/src/components/usage.tsx:20]: I get an error when I do that. Should I leave the fragment tag or do something else? 
CleanShot 2021-08-31 at 15 49 04

File: docs/src/content/components/dropdown.mdx (hunk 5 above)
Comment (sweeting) [reply docs/src/content/components/dropdown.mdx:384]: 👍

File: docs/src/content/components/dropdown.mdx (hunk 4 above)
Comment (sweeting) [reply docs/src/content/components/dropdown.mdx:69]: 👍

File: docs/src/components/usage.tsx (hunk 3 above)
Comment (jaketrent) [reply docs/src/components/usage.tsx:20]: Oh, sorry, yeah, leave it. I didn't see the second `<ul/>`.

**Review by sweeting — APPROVED** [review]
@jaketrent If you're good with this, would you mind merging and then I'l get back to a normal way of working to update the rest of the docs.

**Comment by jaketrent** [comment]
I approve. I will back-merge master, wait for green build and then merge.

**Pull request merged**